website/src/components/CardLevel.vue

```diff
@@ -96,6 +96,10 @@ export default {
       return this.item.hidden === true
     },
 
+    isVerifyQueue() {
+      return document.location.href.includes('tab_verify_queue');
+    },
```

Comment:
I don't really like that this one is based on what's in the address bar, would be better to just get the currently active tab from somewhere else.